src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -19267,6 +19575,58 @@ static void F(B x, C? y)
                 Diagnostic(ErrorCode.WRN_NullReferenceReceiver, "(new[] { y, x })[0]").WithLocation(13, 9));
         }
 
+        [Fact]
+        public void MultipleConversions_ArrayInitializer_ConversionWithNullableOutput()
```

Comment:
>MultipleConversions_ArrayInitializer_ConversionWithNullableOutput [](start = 20, length = 65)

Can [https://github.com/dotnet/roslyn/issues/31718](https://github.com/dotnet/roslyn/issues/31718) be marked as fixed?

Reply:
Thanks. Added to the description